models/errors/validation-errors.ts

```diff
@@ -135,8 +142,21 @@ export const unauthorisedActionIssue: fhir.OperationOutcomeIssue = {
         system: "https://fhir.nhs.uk/R4/CodeSystem/Spine-ErrorOrWarningCode",
         version: "1",
         code: "ACCESS_DENIED",
-        display: "Required access level not met."
+        display: "Access has been denied to process this request"
       }
     ]
   }
 }
+
+export const invalidHeaderOperationOutcome = (headers: Array<string>): fhir.OperationOutcomeIssue => ({
+  severity: "error",
+  code: fhir.IssueCodes.INVALID,
+  details: {
+    coding: [{
+      system: "https://fhir.nhs.uk/R4/CodeSystem/Spine-ErrorOrWarningCode",
+      code: "MISSING_OR_INVALID_HEADER",
+      display: "There is a required header missing or invalid"
```

Comment:
I'd remove the word `required` from here, given that we're also using this error when the header is forbidden

Reply:
copied this from https://fhir.nhs.uk/STU3/ValueSet/Spine-ErrorOrWarningCode-1 but might not have been the right thing to do lol